app/controllers/search_controller.rb

```diff
@@ -7,18 +7,9 @@ def results
                .or(Post.where('body LIKE ?', "%#{search_term}%"))
                .or(Post.where('username LIKE ?', "%#{search_term}%"))
 
-    case params[:feedback]
-    when "true positive"
-      feedback = 1
-    when "false positive"
-      feedback = 2
-    when "true negative"
-      feedback = 3
-    when "needs edit"
-      feedback = 4
-    end
-
-    if feedback.present?
+    feedback = params[:feedback]
+    if feedback.present? && feedback != "-1" # -1 signifies no specific feedback is selected.
+      feedback.to_i
```

Comment:
FYI, I didn't use `to_i` when initialising `feedback` because it seemed to never work with the "Any" option selected, even though it worked in the rails console. I'm guessing it must be some silly error, but it doesn't really matter as string comparisons work just as well.